webapp/src/ts/services/contact-muted.service.ts

```diff
@@ -8,28 +8,31 @@ export class ContactMutedService {
 
   constructor() { }
 
-  getMuted(doc, lineage?) {
+  getMutedParent(doc, lineage?) {
     if (!doc) {
       return false;
     }
 
     if (doc.muted) {
-      return doc.muted;
+      return doc;
     }
 
     if (lineage) {
-      const mutedParent = _find(lineage, (parent) => parent && parent.muted);
-      return !!mutedParent && mutedParent.muted;
+      return _find(lineage, (parent) => parent && parent.muted) || false;
```

Comment:
The `|| false` is unnecessary, right?

```suggestion
      return _find(lineage, (parent) => parent && parent.muted);
```

Reply:
I just wanted consistency of always returning either he muted doc or a boolean. the `find` will return undefined if nothing is found.